src/main/cpp/hw/interfaces/IDragonMotorController.h

```diff
@@ -32,6 +32,7 @@
 // Third Party Includes
 #include <ctre/phoenix/motorcontrol/RemoteSensorSource.h>
 #include <ctre/phoenix/motorcontrol/StatusFrame.h>
+#include <ctre/phoenix/motorcontrol/TalonFXSensorCollection.h>
```

Comment:
interface shouldn't be including specific types inside it (probably shouldn't have the previous 2 ctre includes either but this one really drives at a specific implementation